projects/libvpx/vpx_dec_fuzzer.cc

```diff
@@ -58,7 +58,10 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
   if (file == nullptr) {
     return 0;
   }
-
+  // Ensure input contains at least one file header and one frame header
+  if (size < IVF_FILE_HDR_SZ + IVF_FRAME_HDR_SZ) {
```

Comment:
the read below covers the header minimum, sending in less than expected sizes is a good test for the library

Reply:
If the size is lesser than 32 + 12,  feof check in read_frame fails and returns 1 and breaks the loop below. So no testing would be done in that case as well.